### ISSUE TITLE
Bug in latest 21.06 build 

### ISSUE THREAD

**Issue by Holguitar** [Pittvandewitt/Wavelet#87]
After using the "new" app (UI overhaul, permissions to recognise active audio) for some days on my Xiaomi Mi 9T (MIUI 12/Android 10, main music app is latest Poweramp, also in use is YT vanced and Spotify) I've found some bugs.

First of all I granted the new dump and notification listener permissions and the app was working so far.
After a while the new UI was freezing sometimes, especially after using the power button.
After that two different scenarios appeared:
- No settings are visible but with a delay the app is working as one can hear but without the possibility to change anything.
- On the other hand sometimes the settings are visible but no sound processing is active, no difference to the "normal" sound also not in legacy mode.
It's difficult to get the moment when these bugs appear. I'm back on 21.03 version for the moment (even if I have to dispense with the premium features).
I will update to 21.06 again these days and try to log the whole thing so logs will follow.

**Comment by Coco-Lasticot** [comment]
I have a similar setup:
- Xiaomi Mi 9T Pro (MIUI 12/Android 10)
- latest Poweramp
- Youtube Vanced 
- Spotify
- Granted permissions
with similar effects.

Although "no settings visible" seems to be the everlasting behaviour, now,

Back to 21.03 for me too

**Comment by auanasgheps** [comment]
I can confirm, having the same issue on OnePlus 8T (Android 11)

**Comment by Pittvandewitt** [comment]
The issue with no settings appearing seems to be related to specific apps to which no effects can be attached. When that particular application stops playing music, effects will reapper. Can this be confirmed, and if yes, which audio source seems to be the culprit?

**Comment by Holguitar** [comment]
Sorry for the late reply. I'm on 21.08 (beta channel) now and I can confirm for Poweramp:
If settings disappear they will return when Poweramp is started again.
So everything ok now for the music player.
If I start YouTube vanced (15.33.34, older version for the reason that later builds are not able to cast to Chromecast on non rooted devices) there are no settings at all, but it works on legacy mode.
Have to test Spotify, not installed at the moment.
In summary Wavelet seems to be more stable now. If I find any bugs I will report, if you need something like logs let me know.
For the moment thanks for working on Wavelet.

**Comment by auanasgheps** [comment]
Yeah, for me too. The latest build _seems_ more stable.

**Comment by Pittvandewitt** [comment]
Thank you both for the replies. Feel free to reopen this issue if you still encounter it.
